github/ci/prow/files/jobs/kubevirt/cluster-network-addons-operator/cluster-network-addons-operator-presubmits.yaml

```diff
@@ -130,9 +130,10 @@ presubmits:
         - release-\d+\.\d+
       annotations:
         fork-per-release: "true"
-      always_run: false
+      always_run: true
       optional: true
       decorate: true
+      skip_report: false
```

Comment:
the default value of skip_report is false from what i remember, and it was suggested once that default value field wont be presented